_posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown

```diff
@@ -50,8 +51,11 @@ POST /_reindex
    }
 }
 ```
+Lastly, download the latest version of OpenSearch from the [OpenSearch downloads page](https://opensearch.org/downloads.html). If you need help with installation, see the guidance in [Install and Configure OpenSearch](https://opensearch.org/docs/latest/opensearch/install/index/).
+
+As described in a [previous blog post](https://opensearch.org/blog/technical-posts/2021/07/how-to-upgrade-from-opendistro-to-opensearch/), open-source Elasticsearch is the foundation for Open Distro, and it has the same upgrade paths. You can follow through the steps for the path you choose to upgrade your cluster.
 
-### Support Matrix for open source Elasticsearch to OpenSearch
+## Support matrix for open-source Elasticsearch to OpenSearch
 
 |Open Source Elasticsearch|Recommended Upgrade Path	|
```

Comment:
@saratvemulapalli Do we have any updates to make to this table?

Reply:
@vagimeli this doesnt change. It should be good.